src/humanloop/otel/constants.py

```diff
@@ -0,0 +1,6 @@
+# Attribute name prefix on Humanloop spans for file-related attributes + path
+HL_FILE_KEY = "humanloop.file"
```

Comment:
can we rename these to `HUMANLOOP_...`? i don't see the point of shortening to HL.

Reply:
ack - fixed